### DISCUSSION TITLE
cannot delete .PART file

### DISCUSSION THREAD

**Discussion by stormthedog** [aclap-dev/video-downloadhelper#132] [Help and Q&A]
### Your question

There was an error on completion of a download - No media to process. The two video files created were deleted but the .part file keeps returning. I presume some process is attempting to complete the download. Any help appreciated.

### Addon & CoApp version and details

version: 8.1.4.0
beta: false
build date: 2024-01-04
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-GB
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\name",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by paulrouget** [comment]
The .part is created by Firefox. So close Firefox first?

**Comment by GrampaWildWilly** [comment]
Every once in a while somebody hits a problem vaguely like this.  These are the things to try, in order of ascending severity.

In the file manager, refresh the display.  The file might no longer exist.  Maybe.

Close the browser (like Paul said) & try again.

Do a `chkdsk /f` on the partition in question.  Sometimes, this runs right away.  Sometimes this tells you it will perform the `chkdsk` on the next reboot.  If it does it right away, go back & try to delete the file again.  If it tells you the `chkdsk` will happen on the next reboot, do the reboot whenever is convenient.  Make sure to pay strict attention to what is displayed during the reboot.  Make sure you see messages about the `chkdsk` being performed.  If you don't see the messages during the boot up phase, try again.  Make sure the `chkdsk` is performed.  Then try again to delete the file.

Failing all of that, you'll need to get help from someone more knowledgeable than I am.

**Comment by stormthedog** [comment]
OK. Firefox, open or closed, makes no difference. Chkdsk was performed on boot but the problem remains. If I try to delete using File Shredder, it says I either do not have the necessary privileges, or the file is in use.

If this is not a DownloadHelper issue, I'll look elsewhere.

Thanks for your help.

**Reply by GrampaWildWilly** [reply]
Privileges . . . hmmmm......  What do you have in the Properties dialog for that directory on the Security tab?

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/0c4b550d-4596-41c5-89fd-da830664211c)

Be sure you highlight your user in the Group or user names window in the top half of that dialog.

Also, what do you get for attributes on the directory?  This is from a command prompt.

```
M:\>attrib "VDH Testing"
A                    M:\VDH Testing
```

**Comment by stormthedog** [comment]
Confused by the instructions. However, I compared the values for a similar file. The **differences** are - 
Authenticated Users. Corrupt file = Full Control: **black tick**. Good file = Full Control: **no tick**.
Users. Corrupt file = Full Control, Modify, Write: **black tick**. Good file = Full Control, Modify, Write: **no tick**

Directory attributes -
J:\>attrib "stuff\new\deleteme"
A                    J:\stuff\new\deleteme

**Comment by stormthedog** [comment]
Actually, I might have succeeded in deleting the file without it springing back into life by simply disabling DownloadHelper in Firefox. I re-enabled DownloadHelper and it's still not returned. Hope it stays that way! Thanks for taking the time to look into this. Appreciated.

**Reply by GrampaWildWilly** [reply]
Oh now that is interesting.  Completely closing the browser & rebooting the system didn't let you delete the file.  But keeping the browser open & explicitly disabling the extension somehow unlocked the file & you could delete it.  That seems about as illogical as it can get.  But whatever.  This is going in my bag of tricks for the next person who encounters this problem.  Thanks a bunch!

**Comment by stormthedog** [comment]
No problem. Can confirm it hasn't returned. DownloadHelper clearly was determined to download all of that file!
